tests/client/amo/containers/TestSearch.js

```diff
@@ -23,13 +24,16 @@ describe('Search.mapStateToProps()', () => {
   };
 
   it('passes the search state if the URL and state query matches', () => {
-    const props = mapStateToProps(state, { location: { query: { q: 'ad-block' } } });
+    const props = mapStateToProps(state,
```

Comment:
I really prefer the PEP8 style where all of the arguments need to be at the same indentation level.

```js
const props = mapStateToProps(
  state,
  { location: { query: { q: 'ad-block' } } });
// or
const props = mapStateToProps(
  state, { location: { query: { q: 'ad-block' } } });
// or, preferably not, bot possibly
const props = mapStateToProps(state,
                              { location: { query: { q: 'ad-block' } } });